### PR TITLE
PR: update contributor & concurrently build of frontend and backend (#4)

### PR DESCRIPTION
**Pull request by shinevue**
* fix: update contributor

* fix: concurrently build

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,60 @@
+{
+  "name": "finalgoal",
+  "version": "1.0.0",
+  "description": "Admin Dashboard template built with create-react-app, tailwind css and daisy UI. Template uses rich tailwind css utility classes and have components of daisy UI, also have redux toolkit implemented for store management.",
+  "scripts": {
+    "start": "concurrently \"npm run dev:frontend\"",
```
Comment (Johnhvy) [inline package.json:6]: There isn't any command to run concurrently

**Review by Johnhvy — CHANGES_REQUESTED** [review]

**Review by Johnhvy — APPROVED** [review]

**Review by Johnhvy — APPROVED** [review]
Good job

**Review by shinevue — COMMENTED** [review]
Approve

**Review by creative2113 — APPROVED** [review]
Nice job

**Review by creative2113 — APPROVED** [review]
good style code

**Review by shinevue — COMMENTED** [review]
Thank you

**Pull request closed without merging**